ush/python/pygfs/task/atmens_analysis.py

```diff
@@ -289,17 +300,30 @@ def jedi2fv3inc(self: Analysis) -> None:
             memchar = f"mem{imem:03d}"
 
             # make output directory for member increment
-            incdir = [
-                os.path.join(self.task_config.COMOUT, memchar, 'atmos')
-            ]
-            FileHandler({'mkdir': incdir}).sync()
+            template = self.task_config.COM_ATMOS_ANALYSIS_TMPL
+            tmpl_dict = {
```

Comment:
Since other than `MEMDIR`, these do not change from member to member, can we define two dicts before the loop (one for current cycle and one for previous cycle) and then just change the `MEMDIR` of each inside each iteration?

Reply:
Requested change made at [`37fbac4`](https://github.com/RussTreadon-NOAA/global-workflow/commit/37fbac4bb01b74a9a642d2fbb3a871f2aafc6432)